tests/test_covar2d.py

```diff
@@ -88,7 +88,7 @@ def test02GetCovar(self):
     def test03GetMeanCTF(self):
         results = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_cov2d_meanctf.npy'))
         self.mean_coeff_ctf = self.cov2d.get_mean(self.coeff, self.h_ctf_fb, self.h_idx)
-        self.assertTrue(np.allclose(results, self.mean_coeff_ctf))
+        self.assertTrue(np.allclose(results.flatten(), self.mean_coeff_ctf))
```

Comment:
Same.

Reply:
The saved data was still in 2D array. Updated it .